App/Models/Task.cs

```diff
@@ -16,4 +16,20 @@ public Task(int id, string title, string description, DateTimeOffset created, Da
         Due = due;
         IsDone = isDone;
     }
+    
+    public override bool Equals(object obj)
+    {
+        return obj is Task task &&
+            Id == task.Id &&
+            Title == task.Title &&
+            Description == task.Description &&
+            Created.ToString() == task.Created.ToString() &&
```

Comment:
This used to be a problem. When I convert an object of the DateTimeOffset class to a string, and then back to DateTimeOffset, when comparing the old and new values, it turns out that they are different.
![image](https://github.com/Nemesnyky/studennyk/assets/127392046/cdfc13f5-8870-4aac-845e-583205aa3be9)

So I used this костиль. But I have already found a solution.  Now I'm going to finalize the code so that everything works correctly.